### PR TITLE
all-your-base: Update json for changes to "input" policy

### PR DESCRIPTION
**Pull request by Stargator**
#996 Updated the schema for canonical data. So I am updating the all-your-base exercise's json to reflect the change.

### CODE REVIEW

File: exercises/all-your-base/canonical-data.json [hunk 1]
```diff
@@ -16,169 +16,280 @@
...
     {
       "description": "binary to single decimal",
       "property": "rebase",
-      "input_base": 2,
-      "input_digits": [1, 0, 1],
-      "output_base": 10,
+      "input": {
+        "base": 2,
+        "digits": [
+          1,
```
Comment (ErikSchierboom) [inline exercises/all-your-base/canonical-data.json]: Any specific reason to list each digit on a separate line? Seems to be a little much here (and for the other cases too).

Comment (Stargator) [reply exercises/all-your-base/canonical-data.json]: My IDE automatically formatted it and I figured if that is how JSON is formatted, then I wouldn't object.

File: exercises/all-your-base/canonical-data.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "exercise": "all-your-base",
-  "version": "2.0.1",
+  "version": "2.1.1",
```
Comment (rpottsoh) [inline exercises/all-your-base/canonical-data.json]: I think this should be "2.1.0".

File: exercises/all-your-base/canonical-data.json [hunk 3]
```diff
@@ -16,169 +16,280 @@
...
       "description": "single bit one to decimal",
       "property": "rebase",
-      "input_base": 2,
-      "input_digits": [1],
-      "output_base": 10,
+      "input": {
+        "input_base": 2,
+        "digits": [
+          1
+        ],
```
Comment (rpottsoh) [inline exercises/all-your-base/canonical-data.json]: I don't believe the schema requires arrays to be formatted this way.  It is especially off-putting to see two different style of arrays.  Focus just on creating the "input" object.

File: exercises/all-your-base/canonical-data.json (hunk 2 above)
Comment (Stargator) [reply exercises/all-your-base/canonical-data.json]: Oops, thanks

File: exercises/all-your-base/canonical-data.json (hunk 3 above)
Comment (Stargator) [reply exercises/all-your-base/canonical-data.json]: Gotcha

**Comment by rpottsoh** [comment]
@Stargator merge when ready.

**Pull request merged**